resources/definitions/fdmprinter.def.json

```diff
@@ -2013,6 +2013,29 @@
                     "value": "jerk_layer_0",
                     "enabled": "jerk_enabled",
                     "settable_per_mesh": false
+                },
+                "speed_for_pressure_enabled":
+                {
+                    "label": "Equalize Nozzle Pressure",
+                    "description": "Attempt to dampen nozzle pressure changes by varying printing speed with extrusion width for narrow extrusions",
```

Comment:
Suggestion: 
"label": "Equalize Filament Flow"
"description": "Print thinner than normal lines faster so that the amount of material extruded per second remains the same. Thin pieces in your model might require lines printed with smaller line width than provided in the settings. This setting controls the speed changes for such lines."